script.js

```diff
@@ -0,0 +1,135 @@
+
+const searchButton = document.querySelector('#search-button');
+let searchBar = document.querySelector('#search-input');
+
```

Comment:
recommend using `const` for all variables that aren't reassigned; i.e. selecting elements